fkbeta/fk/migrations/0006_videofile_loudness.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2018-12-01 20:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('fk', '0005_remove_duration_old'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='videofile',
+            name='integrated_lufs',
+            field=models.FloatField(blank=True, null=True, verbose_name=b'Integrated LUFS of file defined in ITU R.128'),
+        ),
+        migrations.AddField(
+            model_name='videofile',
+            name='truepeak_lufs',
+            field=models.FloatField(blank=True, null=True, verbose_name=b'True peak LUFS of file defined in ITU R.128'),
+        ),
+        migrations.AlterField(
+            model_name='video',
+            name='is_filler',
+            field=models.BooleanField(default=False, help_text=b'You still have the editorial responsibility.  Only affect videos from members.', verbose_name=b'Play automatically?'),
```

Comment:
Heh, haven't we seen this many times? But hopefully this'll make it go away now.

Reply:
We sure have.  No idea why, but I removed it from my pull request.